docs/managing_deis/using-a-ldap-auth.rst

```diff
@@ -0,0 +1,48 @@
+:title: Using a LDAP Auth
+:description: How to configure LDAP to Authenticate users.
+
+.. _using-a-ldap-auth:
+
+Using a LDAP Auth
+=================
+
+Deis Controller supports Single Sign On access control, for now Deis is able to authenticate using LDAP or Active Directory.
+
+Settings used by LDAP
+---------------------
+
+=========================================           =================================================================================
+setting                                             description
+=========================================           =================================================================================
+/deis/controller/auth/ldap/endpoint                 The full LDAP endpoint. (Ex.: ldap://ldap.company.com)
+/deis/controller/auth/ldap/bind/dn                  Full user for bind. (Ex.: user@company.com. For Anonymous bind leave blank)
+/deis/controller/auth/ldap/bind/password            Password of the user for bind. (For anonymous bind leave blank)
+/deis/controller/auth/ldap/user/basedn              The BASE DN where your LDAP Users are placed. (Ex.: OU=TeamX,DC=Company,DC=com)
+/deis/controller/auth/ldap/user/filter              The field that we will match with username of Deis. (In most cases is uuid, AD uses sAMAccountName)
+/deis/controller/auth/ldap/group/basedn             The BASE DN where the groups of your LDAP are are located. (Ex.: OU=Groups,OU=TeamX,DC=Company,DC=com)
+/deis/controller/auth/ldap/group/filter             The field that we will locate your groups with LDAPSearch. (In most cases is objectClass)
+/deis/controller/auth/ldap/group/type               The Groups type of LDAP. (Use groupOfNames if you don't know)
+=========================================           =================================================================================
+
+Configuring LDAP on Controller
+------------------------------
+
+.. important::
+
+    It's important that you have a local admin user ( see :ref:`Register a User <register-user>` ) and have disabled the registration ( see :ref:`disable_user_registration` ) in order to only LDAP users and the local admin can login into controller.
```

Comment:
Perfect, I will try to elaborate more about the problems if the users don't do this.